objax/nn/init.py

```diff
@@ -41,6 +41,24 @@ def gain_leaky_relu(relu_slope: float = 0.1):
     return np.sqrt(2 / (1 + relu_slope ** 2))
 
 
+def identity(shape: Tuple[int, ...], gain: float = 1) -> JaxArray:
+    """Returns the identity matrix. This initializer was proposed in
+    `A Simple Way to Initialize Recurrent Networks of Rectified Linear Units
+    <https://arxiv.org/abs/1504.00941>`_.
+
+    Args:
+        shape: Shape of the tensor. It should have exactly rank 2.
+        gain: optional scaling factor.
+
+    Returns:
+        Tensor initialized to the identity matrix.
+    """
+    assert  len(shape) == 2
+    return gain * jn.eye(*shape)
+    return gain * eye
```

Comment:
Shouldn't this return statement be removed?

Reply:
yeah, I just used the commit suggestion from github, guess there was some kind of bug, sorry!